instructors.md

```diff
@@ -3,28 +3,126 @@ layout: page
 title: Version Control with Git
 subtitle: Instructor's Guide
 ---
-## Legend
+Version control might be the most important topic we teach,
+but Git is definitely the most complicated tool.
```

Comment:
Reword this to make clear Git is the most complicated tool *we teach*. As written this means "Git is the most complicated tool *in existence*".